guide/src/componentDocs/ButtonMenu/ButtonMenuDoc.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { P } from 'cyverse-ui';
+import meta from 'cyverse-ui/ButtonMenu.json';
+import { Figure, ComponentDoc, CodeBlock } from '../../components';
```

Comment:
It didn't work... however setting an alias would allow for this. I think it is a good idea in general. `../../` feels wrong.